src/bp-templates/bp-nouveau/includes/media/attachment.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
```

Comment:
```suggestion
/**
 * Media Attachment.
 *
 * @since   BuddyBoss [BBVERSION]
 * @package BuddyBoss\Core
 */

// Exit if accessed directly.
defined( 'ABSPATH' ) || exit;
```